static/css/calendar-icon.less

```diff
@@ -0,0 +1,92 @@
+/* ==========================================================================
+   cfgov-refresh
+   calendar-icon
+   ========================================================================== */
+
+/* topdoc
+  name: Calendar Icon
+  family: cfgov-calendar-icon
+  notes:
+    - "A square-ish calendar icon with a three-letter month name and date."
+  patterns:
+    - name: Calendar Icon
+      markup: |
+        <time class="calendar-icon" data-month="Jan" data-day="21" datetime="2015-01-21">
+          <span class="u-visually-hidden">Jan 21, 2015</span>
+        </time>
+        denotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          time.calendar-icon
+            span.u-visually-hidden
+  tags:
+    - cfgov-calendar-icon
+*/
+
+
+/*  Mixin to generate a calendar icon.
+    @month [String] The month (3-letter abbreviation, typically).
+    @day [Number] The day in the month.
+    @background-color [Color] A color value for the color behind the month.
```

Comment:
Changed to `@background-color [Color] A color value for the icon background (behind the month)`  in https://github.com/cfpb/cfgov-refresh/commit/ec6f6df230e17c72e05be017c8d5f1de3e34d7ac. I just wanted to include the month/day as references so if looking at the icon it would be easy to quickly tell which var was pointing to which color, if that's not already obvious.